source/Plugins.h

```diff
@@ -45,6 +48,17 @@ struct Plugin {
 // Tracks enabled and disabled plugins for loading plugin data or skipping it.
 // This object is updated by toggling plugins in the Preferences UI.
 class Plugins {
+public:
+	struct PluginInstallData
```

Comment:
Just InstallData? It seems redundant to have Plugin twice

Reply:
renamed